deepchem/models/keras_model.py

```diff
@@ -173,7 +173,7 @@ def _create_inputs(self, example_inputs):
       return
     self._ensure_built()
     self._inputs_built = True
-    if len(self.model.inputs) > 0:
+    if (self.model.inputs is not None) and len(self.model.inputs) > 0:
```

Comment:
Is this a change in TF 2.2?  In 2.1 and earlier, if a model doesn't define inputs then its `inputs` field is an empty list rather than None.  If so, search for all uses of `model.inputs` to make sure all of them will handle it correctly.

Reply:
I think it is a change. The `GraphConvKerasModel` class has `model.inputs` set to `None` upon construction. I went through the file and searched for all uses to make sure it was handled correctly.